controllers/acesFile/acesFile.go

```diff
@@ -0,0 +1,45 @@
+package acesFile
+
+import (
+	"bytes"
+	"strconv"
+	"time"
+
+	"github.com/curt-labs/API/helpers/encoding"
+	"github.com/curt-labs/API/helpers/error"
+	"github.com/curt-labs/API/models/acesFile"
+	"github.com/curt-labs/API/models/brand"
+	"github.com/go-martini/martini"
+
+	"net/http"
+)
+
+func GetAcesFile(rw http.ResponseWriter, req *http.Request, enc encoding.Encoder, params martini.Params) string {
+	version := params["version"]
+
+	var brandObj brand.Brand
+	brandID, err := strconv.Atoi(req.URL.Query().Get("brandID"))
```

Comment:
I think what you linked is a little confusing, the datacontext isn't actually being used right there, but in the middle of a different function. Probably should have linked to here:
https://github.com/curt-labs/API/blob/goapi/models/webProperty/webProperty_model.go#L623